src/orga/guidelines.rst

```diff
@@ -164,3 +164,21 @@ The guidance that we can provide here is two fold:
    For example, we're holding off on turning off py36 until pypy comes out with pypy3.7.
 3. If there are lots of people in the community relying on older versions, core team can decide to keep an old version around
    For example, we held off turning off py27 even after numpy, scipy dropped support as there were many in the community interested in keeping support until the end of life of that version.
+
+
+Requesting ``core`` to Merge PRs
+================================
+
+Due to various reasons, it is sometimes necessary for the ``core`` team to merge PRs on feedstocks they do not personally
+maintain. The ``core`` team abides by the following guidelines when doing this.
+
+1. The ``core`` team will normally wait three days before merging a PR on a feedstock they do not maintain personally. This
+   rule encompasses both migration PRs from the autotick bot and requests from community members to merge PRs on feedstocks.
+2. In extenuating circumstances (i.e., bad bugs, blocking lots of other PRs/work, unmaintained feedstocks), the ``core`` team reserves the right to merge
+   before the three day window has passed. In this case, at least two ``core`` members must approve the merge. Approvals include
+   the act of merging, a ``core`` member requesting something be merged, and a ``core`` member making the PR being merged.
+3. The ``core`` team will do its best to not merge PRs marked as in a draft state, either explicitly as a github draft PR or
+   through a label, comment, or indication in a title.
+4. The ``core`` team will do its best to not merge PRs with requested changes (via a comment, label, title, or a PR review),
```

Comment:
There should be a vote to override in this case. PRs shouldn't be mergable just because one core member thinks that they should be. Especially when the issue is contentious and there is disagreement.

Reply:
This clause is meant to apply to PRs on feedstocks-only where a maintainer is completely MIA after asking for changes and work continues on the PR. I am happy to add a voting clause here, but I was hoping we could all be respectful and apply this narrowly.

Reply:
I wish so too, but I think conda-forge is at a scale where intentions need to be spelled out.